https://github.com/json-c/json-c/commit/36ec47db49277f4a5eda30823cf6d1f238f34991

From 36ec47db49277f4a5eda30823cf6d1f238f34991 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Thomas=20Gst=C3=A4dtner?= <thomas@gstaedtner.net>
Date: Sun, 3 Mar 2013 00:17:25 +0100
Subject: [PATCH] configure.in: mv AM_CONFIG_HEADER to AC_CONFIG_HEADER

the former has been deprecated and does not work on newer autoconf
versions.
---
 configure.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.in b/configure.in
index 1a428b6..d68d7a1 100644
--- a/configure.in
+++ b/configure.in
@@ -20,8 +20,8 @@ AM_CONDITIONAL(ENABLE_OLDNAME_COMPAT, [test "x${enable_oldname_compat}" != "xno"
 # Checks for libraries.
 
 # Checks for header files.
-AM_CONFIG_HEADER(config.h)
-AM_CONFIG_HEADER(json_config.h)
+AC_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADER(json_config.h)
 AC_HEADER_STDC
 AC_CHECK_HEADERS(fcntl.h limits.h strings.h syslog.h unistd.h [sys/cdefs.h] [sys/param.h] stdarg.h locale.h)
 AC_CHECK_HEADER(inttypes.h,[AC_DEFINE([JSON_C_HAVE_INTTYPES_H],[1],[Public define for json_inttypes.h])])
-- 
1.8.1.6

